src/katello/repos.py

```diff
@@ -63,6 +64,9 @@ def is_valid(self):
         # wrapper try block w/ finally needed for python 2.4
         try:
             try:
+                if not os.path.exists(ENABLED_REPOS_CACHE_FILE):
```

Comment:
This method already handles the case where the cache file is missing

Reply:
not really .. I  thought it did but it doesnot. OSError is different from IOError

Reply:
Oh, looks like this needs a rebase - master is checking IOError